.github/features_schema.json

```diff
@@ -1,45 +1,63 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "required": ["service", "name", "features"],
+  "required": ["service", "name", "aws_provider","features", "emulation_level"],
   "properties": {
     "service": {
       "type": "string",
-      "description": "Service's abbreviation in AWS"
+      "pattern": "^[A-Za-z0-9]+$",
+      "description": "Short service name"
     },
     "name": {
       "type": "string",
+      "pattern": "^[A-Z][^.\n]*$",
       "description": "The display name of the service from AWS"
     },
+    "aws_provider": {
+      "type": "object",
+      "required": ["api"],
+      "properties": {
+        "api": {
+          "type": "string",
+          "pattern": "^[a-z0-9-_]+$",
+          "description": "Name of AWS API provider or method name"
+        },
```

Comment:
nit: This should theoretically be the same as the short service name above, or did you find instances where they differ?